### DISCUSSION TITLE
Unload settings all metal hotend

### DISCUSSION THREAD

**Discussion by ttheelen** [mriscoc/Ender3V2S1#832] [General]
Hi, 

I'm using an ender 3 V2 with a micro swiss all metal hotend and recently switched to this firmware from Jyers.  Since I'm using the firmware, my hotend is constantly clogging, when the filament is unloaded.  My filament sensor basically became useless and I also can't use filament change (M600) or the firmware's "unload filament" function, because all of those immediately lead to a clogged hotend. 

I know, that micro swiss hotends are notorious for issues like this, but I rarely had problems like this when using jyers and since switching to this firmware, I have to disassemble my hotend almost daily.

I don't have any clogging issues when printing normally, I use a retraction <= 1 mm and everything works fine, it's really the automatic unloading in this firmware, that seems to screw everything up. 

I'm using Cura and Octoprint and use the custom CGode-Scripts found in the wiki.

So, does anyone have experience with issues like this? Are there any settings I can make in the slicer/firmware/octoprint  that are better suited for all metal hotends than the default ones? 

Thank You!

**Comment by mriscoc** [comment]
Try reducing the extruder acceleration and speed.
